### PR TITLE
perf(FiltersPanel): drop extra refetch

### PR DESCRIPTION
**Pull request by KateKate**
## PR includes

- [X] Refactor

### CODE REVIEW

File: src/components/FiltersPanel/FiltersPanel.tsx [hunk 1]
```diff
@@ -33,7 +33,14 @@ import { WatchingFilter } from '../WatchingFilter/WatchingFilter';
...
 const take = 5;
-const queryParams = {
+
+// disable refetchOnMount since we use filtersPanelSsrInit
+const useQueryOptionsRefetchOnMount = {
+    refetchOnMount: false,
+};
+
+const useQueryOptions = {
+    ...useQueryOptionsRefetchOnMount,
```
Comment (9teen90nine) [inline src/components/FiltersPanel/FiltersPanel.tsx:43]: why can't you add `refectOnMount` here without a new variable?

**Review by 9teen90nine — CHANGES_REQUESTED** [review]
Branch and PR titles are wrong

File: src/components/FiltersPanel/FiltersPanel.tsx (hunk 1 above)
Comment (KateKate) [reply src/components/FiltersPanel/FiltersPanel.tsx:43]: To use the same value (w/o copy-paste) for different hooks as options: [useQueryOptions](https://github.com/taskany-inc/issues/pull/1395/files#diff-87ff408285e4f0631d3338be356fce4cd02d84f8cbc7fbe3967af3eb453a9e2eR140) and [useQueryOptionsRefetchOnMount](https://github.com/taskany-inc/issues/pull/1395/files#diff-87ff408285e4f0631d3338be356fce4cd02d84f8cbc7fbe3967af3eb453a9e2eR187).

**Comment by KateKate** [comment]
> Branch and PR titles are wrong

Fixed it - https://github.com/taskany-inc/issues/pull/1409, this PR will be closed.

**Pull request closed without merging**